db/seeds.rb

```diff
@@ -28,3 +28,6 @@
 
 dm = Book.find_by(title: "Children of Dune")
 dm.characters << Character.where(name: CHILDREN_OF_DUNE)
+
+Appearance.of("Chani", in: "Dune").create_arc(summary: "Meets Offworld boy; falls in love; becomes concubine.")
```

Comment:
By virtue of an `Appearance` `has_one :arc` I get `create_arc` [for free](http://guides.rubyonrails.org/association_basics.html#has-and-belongs-to-many-association-reference).

The `Appearance.of`method returns an `Appearance` (the join model instance).